Sadot/User/OrderManagement/OrderForm.cs

```diff
@@ -63,7 +63,7 @@ public OrderForm(Table tableInfo)
 
             linesInExistOrders = new List<LinesInOrder>();
             cancelsInOrder = new List<CancellationsInOrder>();
-    }
+        }
```

Comment:
What is this?

Reply:
Initializations of the lists existed before my changes.
What do you suggest? move these initializations to variable declarations?

Reply:
No, I talked about line 66

Reply:
It looks fine at visual studio I don't know why it's marked as a change (no spaces before the "}")